### PR TITLE
Add MLC, Normalization & Multiple Statuses

### PR DESCRIPTION
**Pull request by Jenarth**
Adding Multi-level classification, Normalization & Multiple Statuses to the python wrapper.

### CODE REVIEW

File: zdai/api/mlcapi.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        Gets the MLC data for the request_id.
+
+        :return:
+        """
+        caller = self._call.new(method = 'GET', path = f'mlc/{request_id}')
+        caller.send()
+
+        return MLCRequest(api = self, json = caller.response.json()), caller
+
+    def get_multiple(self, request_ids: List[str]) -> Tuple[List[MLCRequest]]:
```
Comment (ben741) [inline zdai/api/mlcapi.py:52]:
```suggestion
    def get_multiple(self, request_ids: List[str]) -> Tuple[List[MLCRequest], ApiCall]:
```

File: zdai/api/languageapi.py [hunk 2]
```diff
@@ -48,3 +48,21 @@ def get(self, request_id: str) -> Tuple[LanguageClassificationRequest, ApiCall]:
...
+        caller = self._call.new(method='GET', path=f'languages')
+        caller.add_parameter('request_id', request_ids)
+        caller.send()
+
+        language_requests = []
+
+        for request_id, result in caller.response.json().get('statuses').items():
+            r = result
+            r['request_id'] = request_id
+            language_requests.append(LanguageClassificationRequest(api = self, json = r))
```
Comment (ben741) [inline zdai/api/languageapi.py:66]: I think you can drop the assignment to `r` here (and the other places that follow the same pattern).

```suggestion
            result['request_id'] = request_id
            language_requests.append(LanguageClassificationRequest(api=self, json=result))
```

File: README.md [hunk 3]
```diff
@@ -166,6 +193,40 @@ language, _ = sdk.language.get(request_id = language_jobs[0].id)
...
+DocAI can be used to normalize strings that contain dates, so that the `year`, `month` and `day` are returned.
+
+```python
+from zdai import ZDAISDK
+import json
+
+with open('quick_demo.json', 'r') as file:
+    data = json.loads(file.read())
+    phrases = data.get('normalization')
+    mlc_files = data.get('mlc_files')
```
Comment (ben741) [inline README.md:206]: Looks like some unnecessary code snuck in here.
```suggestion
```

File: zdai/api/mlcapi.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    """
+    MLCAPI contains the functionality accepted by the Multilevel Classification Microservice
+    """
+
+    def __init__(self, token: str, url: str):
+        self._call = ApiCall(token, url)
+
+    def create(self, file_ids: List[str]) -> Tuple[List[MLCRequest], ApiCall]:
+        """
+        Creates a new Language request for the file ids provided.
```
[the remark this answers is not in the document]
Comment (Jenarth) [reply zdai/api/mlcapi.py]: 5f656f1e53c5db6fdba711a10b14bc92a3e4b71e

File: zdai/api/mlcapi.py (hunk 1 above)
Comment (Jenarth) [reply zdai/api/mlcapi.py]: 5f656f1e53c5db6fdba711a10b14bc92a3e4b71e

File: zdai/api/languageapi.py (hunk 2 above)
Comment (Jenarth) [reply zdai/api/languageapi.py]: 5f656f1e53c5db6fdba711a10b14bc92a3e4b71e

File: README.md (hunk 3 above)
Comment (Jenarth) [reply README.md]: 5f656f1e53c5db6fdba711a10b14bc92a3e4b71e

File: zdai/api/mlcapi.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    """
+    MLCAPI contains the functionality accepted by the Multilevel Classification Microservice
+    """
+
+    def __init__(self, token: str, url: str):
+        self._call = ApiCall(token, url)
+
+    def create(self, file_ids: List[str]) -> Tuple[List[MLCRequest], ApiCall]:
+        """
+        Creates a MLC Language request for the file ids provided.
```
Comment (ben741) [inline zdai/api/mlcapi.py:31]:
```suggestion
        Creates an MLC request for the file ids provided.
```

File: zdai/api/fileapi.py [hunk 6]
```diff
@@ -62,3 +64,30 @@ def delete(self, file_id: int) -> Tuple[bool, ApiCall]:
...
         return caller.response.status_code == 204, caller
+
+    def set_expiration(self, file_id: str, expiration: str):
+        """
+        Sets the file expiration
+
+        Example expirations are:
+            2022-12-17T00:00:00Z — sets the expiration to December 17, 2022
+            1d — sets the expiration to 1 day from when this call is run
+            13d — sets the expiration to 13 days from when this all is run
```
Comment (ben741) [inline zdai/api/fileapi.py:75]:
```suggestion
            13d — sets the expiration to 13 days from when this call is run
```

Comment (Jenarth) [reply zdai/api/fileapi.py]: 2efce3cb828dd802d3aeefe942f724bc4fadd0f3

**Review by ben741 — APPROVED** [review]

**Pull request merged**